Sourcery/Parsing/FileParser.swift

```diff
@@ -234,8 +234,25 @@ extension FileParser {
 
     internal func parseVariable(_ source: [String: SourceKitRepresentable], isStatic: Bool = false) -> Variable? {
         guard let (name, _, accesibility) = parseTypeRequirements(source),
-            accesibility != .private && accesibility != .fileprivate,
-            let type = source[SwiftDocKey.typeName.rawValue] as? String else { return nil }
+            accesibility != .private && accesibility != .fileprivate else { return nil }
+
+        var maybeType: String? = source[SwiftDocKey.typeName.rawValue] as? String
```

Comment:
Maybe I'm missing something but that's not the output from `Structure(source)`, it doesn't contain those type information.
We won't support `Foo.makeBar()` for now, this would be pretty hard to support reliably as you can have that in separate file and we are dealing with a single `FileParser`